public-api/src/main/java/org/pmiops/workbench/publicapi/DataBrowserController.java

```diff
@@ -231,6 +289,14 @@
         return ResponseEntity.ok(resp);
     }
 
+    @Override
+    public ResponseEntity<AnalysisListResponse> getConceptAnalysisResults(List<String> conceptIds){
```

Comment:
This isn't quite what i was expecting for a response -- Thinking more like a conceptAnalysis  questionConcept,  where where conceptAnalysis is a object that has conceptId, genderAnalysis, ageAnalysis and thats it . And we get a list of those for each concept we pass.   That said,  this does work for one concept which may be the only way we use it.   

So leave it and let me try it out.